spec/pods.md

```diff
@@ -163,6 +163,7 @@ JSON Schema for the Pod Manifest, conforming to [RFC4627](https://tools.ietf.org
         * **name** (string, optional) name of the image (restricted to [AC Identifier](types.md#ac-identifier-type) formatting)
         * **labels** (list of objects, optional) additional labels characterizing the image
     * **app** (object, optional) substitute for the app object of the referred image's ImageManifest. See [Image Manifest Schema](aci.md#image-manifest-schema) for what the app object contains.
+    * **readOnlyRootFS** (boolean, optional, defaults to "false" if unsupplied) whether or not the root filesystem of the app will be mounted read-only.
```

Comment:
perhaps add an example  readOnlyRootFS: true to the example JSON here? https://github.com/appc/spec/pull/603/files#diff-7bbac9ed3bbf6dbc1688669ce478f5deL113